modules/installation-requirements-user-infra.adoc

```diff
@@ -41,10 +46,27 @@ operating system.
 [id="network-connectivity_{context}"]
 == Network connectivity requirements
 
+ifeval::["{context}" == "installing-ibm-z"]
+All the {op-system-first} machines require network in `initramfs` during boot
+to fetch Ignition config files from the Machine Config Server. The machines are configured with static IP
+addresses. No DHCP server is required.
+endif::[]
+ifeval::["{context}" != "installing-ibm-z"]
 All the {op-system-first} machines require network in `initramfs` during boot
 to fetch Ignition config files from the Machine Config Server. During the initial
 boot, the machines require a DHCP server in order to establish a network
 connection to download their Ignition config files.
+endif::[]
+
+ifeval::["{context}" == "installing-ibm-z"]
+[id="ibm-z-network-connectivity_{context}"]
+== IBM Z network connectivity requirements
+
+To install on IBM Z under z/VM, you require a single z/VM virtual NIC in layer 2 mode. You also need:
+
+*   A direct-attached OSA or RoCE network adapter
```

Comment:
I don't recognize most of the acronyms that you're adding. Please check with the IBM writer and expand whichever ones are not immediately identifiable to Z users.

Reply:
@mariaeis Can you please explain the acronyms used? Thank you!

Reply:
After talking to Andy we would leave it as it is as in this case the expanded acronyms are not self-descriptive either. The "network adapter" in the end of the line should be helpful hint.

Reply:
@zzoubkova I agree it is ok to keep the acronyms